packages/orderbook/src/seaport/components.ts

```diff
@@ -9,3 +10,26 @@ export function getOrderComponentsFromMessage(orderMessage: string): OrderCompon
 
   return orderComponents;
 }
+
+export function getBulkOrderComponentsFromMessage(orderMessage: string): OrderComponents[] {
```

Comment:
Seems like bespoke and error prone logic here? Just observation

Reply:
Yeh undeniably not ideal - but unfortunately there is no real way around it because of how we expose the signable message to callers, rather than using the inbuilt seaport-js methods